.github/workflows/ci.yml

```
@@ -8,6 +8,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - php: '8.0'
+            moodle-branch: 'master'
+            database: 'mariadb'
           - php: '7.4'
             moodle-branch: 'master'
```

I don't know if PHP 8.1 will work yet or what they plan to name the 4.0 branch, but these changes might be nice if we can. I can test it elsewhere to avoid muddying this PR.
```suggestion
          - php: '8.1'
            moodle-branch: 'master'
            database: 'mariadb'
          - php: '8.0'
            moodle-branch: 'MOODLE_400_STABLE'
```